.gitignore

```diff
@@ -47,13 +47,9 @@ Thumbs.db
 # npm packages #
 ####################
 node_modules/
+cfgov/unprocessed/apps/*/node_modules/
 npm-debug.log
 
-# Bower packages #
-##################
-bower_components/
-src/vendor
-
```

Comment:
You'll have some conflicts in this file now.

Reply:
Resolved